compilet-Y7gS9.c

```diff
@@ -0,0 +1 @@
+int boot_compilet() { return 1; }
```

Comment:
This was probably accidentally checked in.

Reply:
Yes, good catch I will fix this.